README.md

```diff
@@ -93,34 +92,34 @@ separate Docker container.
+The vast majority of changes to web pages are not relevant to analysts and we want to avoid presenting those irrelevant changes to analysts at all. It is, of course, not trivial to identify "meaningful" changes immediately, and we expect that analysts will always be involved in a decision about whether some changes are "important" or not. However, as we expand from 10<sup>4</sup> to 10<sup>7</sup> web pages, we need to drastically reduce the number of pages that analysts look at. 
 
 Some examples of **meaningless** changes: 
 - it's not unusual for a page to have a view counter on the bottom. In this case, the page changes **by definition** every time you view it.
 - many sites have "content sliders" or news feeds that update periodically. This change may be "meaningful", in that it's interesting to see news updates. But it's only interesting once, not (as is sometimes seen) 1000 or 10000 times.
 
 An example of a **meaningful** change: 
-- In February, we noticed a systematic replacement of the word "impact" with the word "effect" on one website. This change is very interesting because while "impact" and "effect" have similar meanings, "impact" is a **stronger** word. So there is an effort being made to **weaken** the language on existing sites. Our question is in part: what tools would we need in order to have this change **flagged** by our tools and presented to the analyst as **potentially interesting**?
+- In February, we noticed a systematic replacement of the word "impact" with the word "effect" on one website. This change is very interesting because while "impact" and "effect" have similar meanings, "impact" is a **stronger** word. So, there is an effort being made to **weaken** the language on existing sites. Our question is in part: what tools would we need in order to have this change **flagged** by our tools and presented to the analyst as **potentially interesting**?
 
 ### Sample Data
 
 [`example-data`](./example-data) contains examples of website changes:
 
-- `falsepos-...` files are cases any filter should catch
-- `truepos...` files are cases of changes we care about
+- `false pos-...` files are cases any filter should catch
+- `true pos...` files are cases of changes we care about
 
 This is a small but illustrative sample. Many more samples will be made
 available as soon as possible.
 
 ### Schema
 
 This describes the schema of the SQL databases shared by the Rails app in
-web-monitoring-db and the Python processing bakend in web-monitoring-processing.
+web-monitoring-DB and the Python processing backend in web-monitoring-processing.
```

Comment:
`DB` should actually lower-case here. It’s the name of this GitHub project: https://github.com/edgi-govdata-archiving/web-monitoring-db

(Actually, would you mind making this a link when you lower-case it again? Thanks!)